RusTaxi/Classes/Screens/VC/View Controller/ViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ViewController.swift
+//  RusTaxi
+//
+//  Created by Ruslan Prozhivin on 21.08.2018.
+//  Copyright © 2018 App's ID. All rights reserved.
+//
+
+import UIKit
+import Material
+
+class ViewController: UIViewController, NibLoadable {
+	@IBOutlet weak var tableView: UITableView!
+	
+	private let heightForHeader: CGFloat = 140
+	private let heightForFooter: CGFloat = 112
+	var numberCode: String = "+7"
+	var countryFlag: UIImage = #imageLiteral(resourceName: "ic_flag_russia")
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		delegating()
+		customHeightForHeaderAndFooterView()
+		registerNibs()
+		createNavBar()
+	}
+
+	private func delegating() {
+		tableView.delegate = self
+		tableView.dataSource = self
+	}
+	
+	private func createNavBar() {
+		let navBar: UINavigationBar = UINavigationBar(frame: CGRect(x: 0, y: 20, width: 415, height: 50))
```

Comment:
why you set frame manually and create new bar, instead using existing + magic numbers